tests/system/Commands/CreateDatabaseTest.php

```diff
@@ -25,11 +25,7 @@
 final class CreateDatabaseTest extends CIUnitTestCase
 {
     protected $streamFilter;
-
-    /**
-     * @var BaseConnection
-     */
-    protected $connection;
+    protected BaseConnection $connection;
```

Comment:
the class is final, and parent class doesn't has `$connection` property, so make it typed for protected property.